pkg/storage/cache/filemetadata.go

```diff
@@ -39,5 +39,5 @@ func NewFileMetadataCache(store string, nodes []string, database, table string,
 
 // RemoveMetadata removes a reference from the metadata cache
 func (c *fileMetadataCache) RemoveMetadata(path string) error {
-	return c.Delete(path)
+	return c.s.Delete(path)
```

Comment:
Why do we need this change? `c.Delete` sets `microstore.DeleteFrom` options like the database and table which `c.s.Delete` doesn't. Maybe it's not necessary but then I wonder why `c.Delete` does it in the first place..

Reply:
hm the implementations might forget to use the table and database that was configured when initializing the store. I'll drop this change as it is unrelated.